langchain4j-vertex-ai/src/main/java/dev/langchain4j/model/vertexai/VertexAiEmbeddingModelName.java

```diff
@@ -5,6 +5,11 @@
 
 public enum VertexAiEmbeddingModelName {
 
+    TEXTEMBEDDING_MULTMODAL("multimodalembedding", 1408), // accepts 128, 256, 512 too
```

Comment:
```suggestion
    MULTMODALEMBEDDING("multimodalembedding", 1408), // accepts 128, 256, 512 too
```

Reply:
Not sure about these, I had an impression that we just capitalize the name and replace `-` with `_`?